src/main/java/io/temporal/common/converter/EncodingKeys.java

```diff
@@ -31,7 +31,16 @@
   static final String METADATA_ENCODING_RAW_NAME = "raw";
   static final ByteString METADATA_ENCODING_RAW =
       ByteString.copyFrom(METADATA_ENCODING_RAW_NAME, StandardCharsets.UTF_8);
+
   static final String METADATA_ENCODING_JSON_NAME = "json";
   static final ByteString METADATA_ENCODING_JSON =
       ByteString.copyFrom(METADATA_ENCODING_JSON_NAME, StandardCharsets.UTF_8);
+
+  static final String METADATA_ENCODING_PROTOBUF_JSON_NAME = "protobuf-json";
```

Comment:
Is this the default now?

We want to make sure interop works out of the box between go-sdk and java-sdk.

Reply:
I will make sure that the Go SDK is updated appropriatly